src/lib/read.js

```diff
@@ -0,0 +1,10 @@
+/*
+ * Reads given path and return it's content as utf-8
+ */
+const P = require('bluebird')
+// shall we use something like require fs/promises ?
```

Comment:
Vestigial comment? Or what do you mean?

Reply:
it seems you don't need to promisify the "fs" package as on newer node there is "fs/promises" but i'm not sure, i better delete this indeed